src/Common/Version.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
     <AssemblyVersion>1.13.0</AssemblyVersion>
-    <PackageVersion>$(AssemblyVersion)-next-1</PackageVersion>
-    <DriverVersion>1.12.0-1623185968000</DriverVersion>
+    <PackageVersion>$(AssemblyVersion)-next-2</PackageVersion>
```

Comment:
does this mean we publish a new beta version?

Reply:
Yeah, it'd be ready for that. But that's a manually triggered process.